src/Symfony/Component/ExpressionLanguage/TokenStream.php

```diff
@@ -61,7 +61,7 @@ public function next()
      * @param array|int   $type    The type to test
      * @param string|null $message The syntax error message
      */
-    public function expect($type, string $value = null, string $message = null)
+    public function expect(array|int $type, string $value = null, string $message = null)
```

Comment:
Is this correct? `$type` is passed as 1st argument to `Token::test()` (line 67). That method was updated in this PR to `test(string $type, string $value = null)`. So either this needs to be `string`, or that method needs to be `array|int $type`.